dask/array/creation.py

```diff
@@ -64,13 +66,17 @@ def empty_like(a, dtype=None, chunks=None):
 
     a = asarray(a, name=False)
     return empty(
-        a.shape,
+        (shape or a.shape),
```

Comment:
The shape of `()` is a valid user input that we should probably respect. 
```suggestion
        (shape if shape is not None else a.shape),
```

Reply:
@andersy005 it looks like this change is still needed.